pkg/api/bytes.go

```diff
@@ -36,9 +36,18 @@ func (s *Service) bytesUploadHandler(w http.ResponseWriter, r *http.Request) {
 
 	putter, wait, err := s.newStamperPutter(r)
 	if err != nil {
-		logger.Debug("bytes upload: get putter failed", "error", err)
-		logger.Error(nil, "bytes upload: get putter failed")
-		jsonhttp.BadRequest(w, nil)
+		logger.Debug("bytes upload: putter failed", "error", err)
+		logger.Error(nil, "bytes upload: putter failed")
+		switch {
+		case errors.Is(err, storage.ErrNotFound):
```

Comment:
The  `storage.ErrNotFound` is irrelevant here because the `newStamperPutter` method does not return it. Also, check for other errors returned by the `newStamperPutter`. The same is in the other places where the `newStamperPutter` is used.